src/integrators/sdirk_integrators.jl

```diff
@@ -224,15 +223,15 @@ end
     W = I - dto2*J
   else
     J = ForwardDiff.derivative(uf,uprev)
-    W = 1 - dto2*J
+    W = @. 1 - dto2*J
```

Comment:
Just to be sure: does not the same problem occur in lines https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/integrators/rosenbrock_integrators.jl#L246-L252? Shouldn't we use the identity matrix instead of 1 in line 252 if `J` is an `AbstractArray`?